devel/roswell/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        roswell roswell 21.06.14.110 v
+revision            0
+
+categories          devel
+license             MIT
+maintainers         {@kakuhen} openmaintainer
+platforms           darwin
+supported_archs     x86_64 arm64
+description         A Lisp implementation manager, launcher, and much more!
+long_description    \
+    Roswell is a full-stack environment for Common Lisp development, and has \
+    many features that makes it easy to test, share, and distribute your Lisp\
+    applications.
+
+depends_build       port:autoconf port:automake port:curl
+depends_lib         port:curl
+
+checksums ${name}-${version}${extract.suffix} \
```

Comment:
Just as a note: if your Portfile has checksums for only a **single** file, you do not necessarily have to specify the filename pattern in the checksum block.  

Per the above for this Portfile, the following format would be sufficient:

```
checksums           rmd160  08b5b8dd998f2b016132ed1d73d62933bfb4929a \
                    sha256  ae84150bf2b012dcca5ceced252bedded61ee14156686bbb3d1e38dea955fbd2 \
                    size    168112
```

Reply:
Thanks for letting me know. I have adjusted the Portfile accordingly.